ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -69,6 +69,10 @@ tests:
   commands: make verify
   container:
     from: src
+- as: verify-bundle
+  commands: make verify-bundle
```

Comment:
This is failing rehearsal jobs because operator-sdk is getting called with a version string that doesn't match semantic versioning:

```
 /go/src/github.com/openshift/file-integrity-operator/build/kustomize build config/manifests | ./build/operator-sdk generate bundle -q --extra-service-accounts file-integrity-daemon --overwrite --version 1.22  --default-channel="alpha"
Error: invalid command options: 1.22 is not a valid semantic version: No Major.Minor.Patch elements found 
```

Looks like the golang version and not the operator version, which I think is what we're intending.